test/BUILD.gn

```diff
@@ -256,20 +256,21 @@ test("brave_unit_tests") {
   deps += [
     "//brave/browser/safebrowsing",
   ]
+}
+}
 
-  include_dirs = []
-
-  if (is_win) {
-    sources += [
+if (is_win) {
+  test("brave_install_static_unittests") {
```

Comment:
@bsclifton CI should run one more test target(`brave_install_static_unittests`) only for Windows.

Reply:
can't this be a win only dep of the main test target?

Reply:
or add the sources and deps inside `is_win`? I don't want to add another test target

Reply:
There was a resource id conflicts between `mini_installer` and `chrome_dll`.
This could happen because both are not built together in upstream.
See https://bravesoftware.slack.com/archives/C7VLGSR55/p1561749323241500

Reply:
brave-browser should be updated to run this along with the other unit tests. We can't keep adding test targets and expect that they will actually get run by people

Reply:
ok, will introduce `brave_installer_unittests` target on all platform and only Win has actual test and others are empty target.

Reply:
the important part is that they get run along with existing unit tests

Reply:
@bridiver By modifying `test.js`, running new test target together within `npm run brave_unit_tests` would be possible. One thing it will break is output result. `--output brave_unit_tests.xml` This output file will be overwritten when second test is run.

Reply:
so specify a different output file?

Reply:
That means we need to add one more step in Builder pipeline.
If so, new test target will be run separately?

Reply:
@bridiver Pushed two commits to b-b and b-c. PTAL.
b-b: https://github.com/brave/brave-browser/pull/5085/commits/958d07c48707ec927e6afa7e5aa94157b2a6164d
b-c: https://github.com/brave/brave-core/pull/2829/commits/d57536a273120a3f03ef47daf6cc38764162e635